src/Components/Artwork/GridItem.tsx

```diff
@@ -28,11 +28,52 @@ interface Props extends RelayProps, React.HTMLProps<ArtworkGridItemContainer> {
   }
 }
 
-class ArtworkGridItemContainer extends React.Component<Props, null> {
+interface State {
+  width: number
+  height: number
+}
+
+const GEMINI_HOST = "d7hftxdivxxvm.cloudfront.net"
+const IMAGE_QUALITY = 80
+
+class ArtworkGridItemContainer extends React.Component<Props, State> {
   static defaultProps = {
     useRelay: true,
   }
 
+  image: HTMLImageElement = null
+
+  state = {
+    width: 0,
+    height: 0,
+  }
+
+  componentDidMount() {
+    const { width } = this.image
+    const height = this.image.width * this.props.artwork.image.aspect_ratio
+
+    this.setState({
+      width,
+      height,
+    })
+  }
+
+  get imageURL() {
+    const imageURL = this.props.artwork.image.url
+    if (imageURL) {
+      // Either scale or crop, based on if an aspect ratio is available.
+      const type = this.props.artwork.image.aspect_ratio ? "width" : "fill"
```

Comment:
This line in Emission https://github.com/artsy/emission/blob/27eb8294373be344e4fc2b28675cece224efda93/src/lib/Components/OpaqueImageView.tsx#L85 uses `fit` instead of `width`. Intentional?

Reply:
Yup, width scales the image up a bit to account for vertically oriented images. At `fit` the quality is quite a bit worse with no real gain.

Reply:
@alloy - the best way to expand is to compare two images: 

**Fill**: 
<img width="885" alt="screen shot 2018-07-30 at 4 14 15 pm" src="https://user-images.githubusercontent.com/236943/43428552-ad70b5fe-9413-11e8-9467-da63a7d77415.png">

**Width**: 
<img width="897" alt="screen shot 2018-07-30 at 3 42 59 pm" src="https://user-images.githubusercontent.com/236943/43428563-bbf44d48-9413-11e8-86df-4492d1927dfa.png">

Width returns the "correct" resolution out of all of the presets [listed here](https://github.com/artsy/gemini/blob/7c483f05007c725ac71013a28e47000ce17b8154/app/util/image_resizing.rb#L9)

Reply:
Hrmm. @mzikherman do you know what’s happening here?

Reply:
Ah, addressed in https://github.com/artsy/reaction/pull/1063#discussion_r206525766